plugins/generator-1.20.1/forge-1.20.1/procedures/spawn_entity.java.ftl

```diff
@@ -0,0 +1,8 @@
+<#include "mcelements.ftl">
+<#assign entity = generator.map(field$entity, "entities", 1)!"null">
+<#if entity != "null">
+if (world instanceof ServerLevel _level) {
+	Entity entityToSpawn = ${entity}.spawn(_level, ${toBlockPos(input$x,input$y,input$z)}, MobSpawnType.MOB_SUMMONED);

```

Comment:
Calls to this method are Nullable, so you should perform null checks for calls to entityToSpawn, applies to all 3 files